data/en/test_calculated_summary.json

```diff
@@ -59,7 +59,7 @@
                         "type": "General",
                         "answers": [{
                                 "id": "second-number-answer",
-                                "label": "Second answer in currency total",
+                                "label": "Second answer in currency label",
```

Comment:
I don't understand why this is changed?

Reply:
To make it super obvious that it's the label of the field.  The whole point of my changes was to make sure it was the label of the question, rather then the title that was being displayed on the calculated summary screen.  
This way I can make it explicit that this bit of text is definitely the label and test for it.
That seem reasonable?

Reply:
but "Second answer in currency total" was a unique value on that page? It doesn't really matter, i just couldn't see the reason behind it.